### PR TITLE
Fix/55926 remove premium references

### PR DESCRIPTION
**Pull request by pbking**
Remove references to "Premium Themes".

Changes many locations in the application.

Addresses #55926

### CODE REVIEW

File: client/lib/plans/features-list.js [hunk 1]
```diff
@@ -228,7 +227,7 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'All Premium features' ),
 		getDescription: () => {
 			return i18n.translate(
-				'Including unlimited premium themes, advanced design and monetization options, Pay with PayPal buttons, and a custom domain name for one year.'
+				'Including advanced design and monetization options, Pay with PayPal buttons, and a custom domain name for one year.'
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:230]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Including unlimited premium themes, advanced design and monetization options, simple payment buttons, and a custom domain name for one year.' )` *ES Score: 6*

File: client/lib/plans/features-list.js [hunk 2]
```diff
@@ -949,7 +939,7 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'All Business features' ),
 		getDescription: () =>
 			i18n.translate(
-				'Including the ability to upload plugins and themes, priority support, advanced monetization options, and unlimited premium themes.'
+				'Including the ability to upload plugins and themes, priority support and advanced monetization options.'
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:942]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/themes/single-site-jetpack.jsx [hunk 3]
```diff
@@ -59,9 +59,9 @@ const ConnectedSingleSiteJetpack = connectOptions( ( props ) => {
 			{ ! requestingSitePlans && currentPlan && ! isPartnerPlan && (
 				<UpsellNudge
 					forceDisplay
-					title={ translate( 'Get unlimited premium themes' ) }
+					title={ translate( 'Upload your own themes' ) }
 					description={ translate(
-						'In addition to our collection of premium themes, get comprehensive WordPress' +
+						'In addition to uploading your own themes, get comprehensive WordPress' +
```
Comment (a8ci18n) [inline client/my-sites/themes/single-site-jetpack.jsx:64]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'In addition to our collection of premium themes, get comprehensive WordPress security, real-time backups, and unlimited video hosting.' )` *ES Score: 7*

File: packages/calypso-products/src/plans-list.js [hunk 4]
```diff
@@ -723,8 +723,7 @@ const getJetpackBusinessDetails = () => ( {
 		].includes( plan ),
 	getDescription: () =>
 		i18n.translate(
-			'{{strong}}Best for organizations:{{/strong}} The most powerful WordPress sites: real-time backups ' +
-				'and unlimited premium themes.',
+			'{{strong}}Best for organizations:{{/strong}} The most powerful WordPress sites.',
```
Comment (a8ci18n) [inline packages/calypso-products/src/plans-list.js:717]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( '{{strong}}Best for organizations:{{/strong}} The most powerful WordPress sites: real-time backups and unlimited premium themes.' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/55926-remove-premium-references)

**Comment by pbking** [comment]
This change effects #55941 in that it (currently) changes the wording to exclude premium themes.  It doesn't remove the element.  Would it be best for that whole block to be removed?

**Comment by scruffian** [comment]
I was getting some errors in signup so I removed the other references to FEATURE_PREMIUM_THEMES and that seemed to fix it.

**Comment by scruffian** [comment]
This is looking good to me. I think it deals with all the cases in #55926, but we need the tests to pass before we merge it.

**Review by scruffian — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6540172

Hi @pbking, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Including advanced design and monetization options, Pay with PayPal buttons, and a custom domain name for one year.`
- `Including the ability to upload plugins and themes, priority support and advanced monetization options.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6540172).

**Pull request merged**